java10/session-1-Local-Variable-Type-Inference/02_Object_Data_Types/src/com/typeinference/ObjectTypes.java

```diff
@@ -0,0 +1,52 @@
+package com.typeinference;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ObjectTypes {
+
+	public static void main(String[] args) {
+
+		Integer years = 37; // this object var declaration should be changed
+		String gender = "FEMALE"; // this object var declaration should be changed
+
+		if ( gender.equals( "MALE" ) ) {
+			System.out.println( "Hi Mister, you're " + years + " old " );
+		}
+		else if ( gender.equals( "FEMALE" ) ) {
+			System.out.println( "Hi Miss, you're " + years + " old " );
+		}
+		else {
+			System.out.println( "Hi, you're " + years + " old " );
+		}
+
+		Double totalPayment = 6200D; // this object var declaration should be changed, it might be necessary to make a little change
+		Long days = 85L; // this object var declaration should be changed, it might be necessary a little trick
+
+		Double value = totalPayment / days;  // this object var declaration should be changed
+
+		System.out.println( "Last month you earned $ " + value + " per day " );
+
+		Book theLordOfTheRings = new Book("The Lord Of The Rings", 19.99);
+		Store centralPark = new Store( "Central Park");
+
+		System.out.println( "Your favorite book " + theLordOfTheRings + ", is available in our store " + centralPark.getName() );
+
+
+		List<String> animalss = Arrays.asList( "DOG", "CAT", "MOUSE");
+
+		System.out.println(animalss);
+
+
+		//Checking objects' class
```

Comment:
Remove commented code for now we can always recover it from the VCS

Reply:
@neomatrix369 I let `System.out.println( "animalss is an instance of " + animalss.getClass() );` and added this `System.out` to the other class, it is to see the var type inference

Reply:
So do you want to keep the commented code? uncomment it or remove it, one of the two but not commented code, does not reflect well for an exercise.